src/pages/PageCallParks/index.tsx

```diff
@@ -71,7 +62,43 @@ class PageCallParks extends React.Component<{
             </RnTouchableOpacity>
           ))}
         </ScrollView>
-      </CustomGradient>
+      </>
+    )
+  }
+
+  render() {
+    const authStore = getAuthStore() || {}
+    const { currentProfile } = authStore || {}
+    const { parks } = currentProfile || {}
+    const { selectedPark } = this.state
```

Comment:
Destructure this like const { authStore: { currentProfile: parks } } = getAuthStore() || {} and use parks directly.